uaclient/security.py

```diff
@@ -913,6 +919,53 @@ def _check_subscription_for_required_service(
     return False
 
 
+def _prompt_for_new_token(cfg: UAConfig) -> bool:
```

Comment:
What do you think: Is it worth sharing logic between _prompt_for_new_token and _prompt_for_attach?

Reply:
I think the only part of the code we could reuse is this one:

```
print(status.colorize_commands([["ua", "attach", token]]))
return bool(
    0    
    == cli.action_attach(
        argparse.Namespace(token=token, auto_enable=True), cfg
    )    
)  
```

I think we can move it to a separate function. Let me do that